package.json

```diff
@@ -1137,6 +1159,11 @@
           "type": "boolean",
           "description": "%mssql.ignorePlatformWarning%",
           "default": false
+        },
+        "mssql.objectExplorer.groupBySchema": {
+          "type": "boolean",
+          "description": "%mssql.objectExplorer.groupBySchema%",
+          "default": true
```

Comment:
Are we sure we want to default to true? That seems like it would just annoy people who don't care/don't want it grouped by schema. In general we should probably strive to not change existing defaults unless there's a really good reason to.

Reply:
agreed, default should be false.